code/game/objects/effects/landmarks/landmarks.dm

```diff
@@ -169,6 +173,23 @@
 
 #undef MAXIMUM_LIZARD_AMOUNT
 
+/obj/effect/landmark/xeno_npc_spawn
+	name = "xeno npc spawn"
+	icon_state = "xeno_spawn"
+
+//obj/effect/landmark/xeno_npc_spawn/Initialize(mapload, ...) (removing code for testing reasons)
+//	. = ..()
+//	if(prob(40))
+//		new /mob/living/simple_animal/hostile/alien(loc)
+
+/obj/effect/landmark/xeno_npc_spawn/high_chance_spawn
+
+/obj/effect/landmark/xeno_npc_spawn/high_chance_spawn/Initialize(mapload, ...)
+	. = ..()
+	if(prob(80))
+		new /mob/living/simple_animal/hostile/alien(loc)
+
+
```

Comment:
Make the spawn chance a variable on the landmark itself and you won't need to override it in procs.

Reply:
See above, I'm using what other people have used and a solution is beyond my capabilities.